src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2501,6 +2501,50 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/definitions':
+    get:
+      x-alfresco-since: "6.2.2"
+      tags:
+        - nodes
+      summary: List definitions
+      description: |
+        **Note:** this endpoint is available in Alfresco 6.2.2 and newer versions.
+
+        Retrieve the list of definitions for the node **nodeId**.
+        The list aims to give details about all properties and constraints applied to the node **nodeId** based on its type and aspects.
+        The service will return properties referring to the type hierarchy.
+
+        The default sort order for the returned list of definitions is id ascending.
+        You can override the default by using the **orderBy** parameter.
+
+        You can use any of the following fields to order the results:
+        * id
+        * title
+      operationId: listNodeDefinitions
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/orderByParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/DefinitionPaging'
```

Comment:
As mentioned in the previous comment this should be a singular object, in fact it should probably very similar (if not the same) to the #/definitions/TypeDefinition object you described last week.